src/server/saved_objects/saved_objects_mixin.js

```diff
@@ -57,6 +57,8 @@ export function savedObjectsMixin(kbnServer, server) {
     },
   };
 
+  const schema = new SavedObjectsSchema(kbnServer.uiExports.savedObjectSchemas);
```

Comment:
Mind putting this back where it was? I think it makes more sense to have it next to the serializer and not above the routes.

Reply:
Sure, thing.